crypto/keys/mintkey/mintkey.go

```diff
@@ -42,36 +44,58 @@ var BcryptSecurityParameter = 12
 
 // Armor the InfoBytes
 func ArmorInfoBytes(bz []byte) string {
-	return armorBytes(bz, blockTypeKeyInfo)
+	header := map[string]string{
```

Comment:
Why this change and does it break existing armored keys? What does the version signify? 

EDIT: Looking at `UnarmorInfoBytes`, why not just check the `type`?

Reply:
Will not break existing keys, as they use the old version, and importing supports both old and new.

Feels like version is a better thing to be checking off of.  Isn't that the point of it, so as we upgrade the armored bytes, we use it as a versioning method?  Let's say we want to do another breaking change again, but keep the `type` name the same, we need to switch on version number.

But now that I mention it, I should probably change the new one to be `0.1.0` instead of `0.0.1`.

Reply:
But now that I mention it, should I change the new one to be 0.1.0 instead of 0.0.1.

Reply:
Ok, I see. I would recommend using constants for the header keys though. Also, `0.0.1` is fine for the pub key armor.